db/seeds.rb

```diff
@@ -14,51 +10,241 @@
 Item.destroy_all
 User.destroy_all
 
+options_hash = {headers: true, header_converters: :symbol, converters: :numeric}
+
+# User creation (80 active, 20 inactive)
+users = CSV.open('db/data/MOCK_DATA_users.csv', options_hash)
+user_hashes = users.map { |row| row.to_h }
+n = 1
+
+users = Hash.new
+user_hashes.each do |user_hash|
+  user_hash[:password] = 'password'
+  user_hash[:created_at] = rand(30.0..100.0).days.ago
```

Comment:
I like this touch a lot!

Reply:
Yea, I didn't like that all the dates were today.